pkgs/build-support/kernel/compress-firmware.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i python3 -p "python3.withPackages(ps: with ps; [ click ])"
+import lzma
+import multiprocessing
+import os
+import pathlib
+import shutil
+import typing
+
+
+import click
+
+
+def compress_one(source: pathlib.Path, destination: pathlib.Path):
+    destination.parent.mkdir(parents=True, exist_ok=True)
+    with source.open("rb") as src, lzma.open(destination, "wb") as dst:
```

Comment:
How do we replicate the compression parameters we used before?  Apparently there was some reason to use those particular ones.

Reply:
I don't actually know where we got those, because upstream definitely doesn't use them: https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/tree/copy-firmware.sh#n32

Reply:
Arch I tihnk